### ISSUE TITLE
getPlayerStats placements

### ISSUE THREAD

**Issue by coonmoo** [apugh/realm-api-docs#31]
Hello,

I noticed you changed the "placements" property in the proposal to a single object structure:
`{"1":54,"2":4,"3":21,"4":6,"5":12,"6":7,"7":6,"8":9,"9":2,"11":3,"12":4,"13":4,"14":4,"15":2,"16":3,"17":2,"18":2,"19":4,"20":5,"21":3,"22":2,"23":2,"24":1,"32":1,"35":1,"36":1,"38":1,"46":1,"56":1,"100":7}",`

Currently the API returns an array of objects:

```
[{
"occurrences":54,
"position":"1"
},
{
"occurrences":4,
"position":"2"
}]

```

Is there any reason for this change? 
I would prefer the array of object ways (like it is now), because it expresses more clearly what is meant if you provide users with a library (npm or nuget e.g.).

E.g. if we publish a nuget library and we just have one object (like in the proposal) the property would look like this in C#:

`public Dictionary<string, int> Placements {get;set;}`

but the library user wouldn't know what is the string and what does the int mean?

If expressed as an array of objects its more clearly:

```
public List<Placement> Placements {get;set;}

    public class Placement
    {
        public int occurrences { get; set; }

        public string position { get; set; }
    }

```

Of course repeating the property names adds some more bytes to the response (redundancies), but this could be e.g. neglected by adding gzip compression to the response.

**Comment by apugh** [comment]
The initial implementation included the **Placements** array of objects.

However, this didn't match the proposed documentation so I _additionally_ returned **placement_list** as a string.

Both (should) return identical information (just packaged differently).

Obviously this is a redundancy ... I believe I asked which is preferred in hope of eliminating one or the other.  But I'll probably keep both around for a while since the placement_list isn't really that resource intensive.

**Comment by coonmoo** [comment]
Sounds good. I'd prefer the array of objects

**Comment by fossilz** [comment]
Can you adjust the placement_list so it returns an actual object rather than a string? 

I.E.
Currently
```json
"placement_list": "{\"1\":8, \"2\":5}"
```
Should be 
```json
"placement_list": {"1":8, "2":5}
```

**Comment by apugh** [comment]
Yes, I'll look into this on Monday if nobody objects between now and then.

**Comment by apugh** [comment]
This is proving more difficult than I anticipated and I have other priorities for the next couple weeks.

Use the **placements** object in the meantime ... the data is the same as that contained in **placement_list** string.

**Comment by fossilz** [comment]
No problem. I understand
